files/bootstrap.sh

```diff
@@ -589,8 +589,10 @@ if command -v nvidia-smi &> /dev/null; then
     echo $GPUNAME
 
     # set application clock to maximum
```

Comment:
Can you add a comment here referencing the doc you linked?

Reply:
Good point, addressed in https://github.com/awslabs/amazon-eks-ami/pull/1097/commits/46eb61c3dcf75f735b34f0906dad8f04aa2d4e8a.